Integrations/VulnDB/VulnDB.py

```diff
@@ -117,41 +122,32 @@ def vulndb_vulnerability_to_entry(vuln):
 def vulndb_vulnerability_results_to_demisto_results(res):
     if 'error' in res:
         return_error(res['error'])
+    if 'vulnerability' in res:
+        results = [res['vulnerability']]
+    elif 'results' in res:
+        results = res['results']
     else:
-        if 'vulnerability' in res:
-            results = [res['vulnerability']]
-        elif 'results' in res:
-            results = res['results']
-        else:
-            demisto.results({
-                'Type': entryTypes['error'],
-                'Contents': res,
-                'ContentsFormat': formats['json'],
-                'HumanReadable': 'No "vulnerability" or "results" keys in the returned JSON',
-                'HumanReadableFormat': formats['text']
-            })
-            return
-
-        for result in results:
-            ec = {
-                'VulnDB': vulndb_vulnerability_to_entry(result)
-            }
-
-            human_readable = tableToMarkdown(f'Result for vulnerability ID: {ec["VulnDB"]["Vulnerability"]["ID"]}', {
-                'Title': ec['VulnDB']['Vulnerability']['Title'],
-                'Description': ec['VulnDB']['Vulnerability']['Description'],
-                'Publish Date': ec['VulnDB']['Vulnerability']['PublishedDate'],
-                'Solution Date': ec['VulnDB']['Vulnerability']['SolutionDate']
-            })
+        return_error('No "vulnerability" or "results" keys in the returned JSON')
+        return
+    for result in results:
+        ec = {
+            'VulnDB': vulndb_vulnerability_to_entry(result)
+        }
+        human_readable = tableToMarkdown(f'Result for vulnerability ID: {ec["VulnDB"]["CvssMetrics"][0]["Id"]}', {
+            'Title': ec['VulnDB']['Vulnerability']['Title'],
+            'Description': ec['VulnDB']['Vulnerability']['Description'],
+            'Publish Date': ec['VulnDB']['Vulnerability']['PublishedDate'],
+            'Solution Date': ec['VulnDB']['Vulnerability']['SolutionDate']
+        })
 
-            demisto.results({
-                'Type': entryTypes['note'],
-                'Contents': res,
-                'ContentsFormat': formats['json'],
-                'HumanReadable': human_readable,
-                'HumanReadableFormat': formats['markdown'],
-                'EntryContext': ec
-            })
+        demisto.results({
+            'Type': entryTypes['note'],
+            'Contents': res,
+            'ContentsFormat': formats['json'],
+            'HumanReadable': human_readable,
+            'HumanReadableFormat': formats['markdown'],
+            'EntryContext': ec
+        })
```

Comment:
```suggestion
        return_outputs(readable_output=human_readable, outputs=ec, raw_response=res)
```
You can replace the `demisto.results()` code block with the above.